lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -44,12 +44,22 @@ class DescribeClass < Cop
 
         def_node_matcher :shared_group?, SharedGroups::ALL.block_pattern
 
-        def on_top_level_describe(node, args)
+        def on_top_level_describe(node, (first_arg, _))
           return if shared_group?(root_node)
           return if valid_describe?(node)
           return if describe_with_rails_metadata?(node)
+          return if string_constant_describe?(first_arg)
 
-          add_offense(args.first)
+          add_offense(first_arg)
+        end
+
+        private
+
+        def string_constant_describe?(first_arg)
+          value =
+            first_arg.lvasgn_type? ? first_arg.child_nodes.first : first_arg
+
+          value.str_type? && value.value =~ /(?:[A-Z]\w*(?:::)?)*[A-Z]\w*/
```

Comment:
This regexp is passing for `activeRecord` (unless it's too late to do code reviews).

Reply:
Thank you! `^` and `$` have been added.